package.json

```
@@ -79,17 +79,31 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-prettier": "2.6.0",
-    "husky": "^0.14.3",
+    "husky": "4.0.10",
     "lint-staged": "^7.0.4",
     "loader.js": "^4.7.0",
     "prettier": "^1.12.0",
     "qunit-dom": "^0.8.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```

```suggestion
    "node": "10.* || >= 12.*"
```

Node 8.x is not an active LTS anymore

---

Changed